plugins/kubernetes/test/models/kubernetes/resource_test.rb

```diff
@@ -805,5 +806,18 @@ def deployment_stub(replica_count)
         end
       end
     end
+
+    describe "#revert" do
+      it "deletes and then creates without resourceVersion because that is not allowed" do
+        with = ->(request) { request.body.wont_include "resourceVersion"; true }
+        assert_request(:get, url, to_return: [{body: '{}'}, {status: 404}]) do
+          assert_request(:delete, url, to_return: {body: '{}'}) do
+            assert_request(:post, File.dirname(url), with: with, to_return: {body: '{}'}) do
```

Comment:
i.e. would like to see this be a variable like create_url - that way i at least can track what's going on

Reply:
dried it up a bit